userspace/libscap/scap.c

```diff
@@ -323,7 +323,16 @@ scap_t* scap_open_live_int(char *error, int32_t *rc,
 		//
 		// Allocate the device descriptors.
 		//
-		len = RING_BUF_SIZE * 2;
+
+		FILE * fp = fopen("/sys/module/" PROBE_NAME "/parameters/ring_buf_size", "r");
```

Comment:
Should probably fix the other `fopen` using `"/sys/module/" PROBE_DEVICE_NAME "_probe/parameters/"` as well. (They all assume PROBE_NAME has the "-probe" suffix, falco doesn't, for example)
The `KMOD_NAME`-like variable could be used here